library/src/main/java/com/getstream/sdk/chat/rest/response/ChannelState.java

```diff
@@ -254,6 +254,7 @@ public String getChannelNameOrMembers() {
             List<User> top3 = users.subList(0, Math.min(3, users.size()));
             List<String> usernames = new ArrayList<>();
             for (User u : top3) {
+                if(u == null) continue;
```

Comment:
I think it would be better to check null for `users`

Reply:
Users are returned from `getOtherUsers`. And it can't return null.
And we have a stacktrace, and it points at `usernames.add(u.getName())`